ci/windows/test_cccl_c_parallel.ps1

```
@@ -16,12 +19,12 @@ If($CURRENT_PATH -ne "ci") {
 }
 
 # Build first
-$buildCmd = "$PSScriptRoot/build_cccl_c_parallel.ps1 -arch '$CUDA_ARCH' -cmake-options '$CMAKE_OPTIONS'"
+$buildCmd = "$PSScriptRoot/build_cccl_c_parallel.ps1 -arch '$CUDA_ARCH' -cmake-options '$CMAKE_OPTIONS' -ENABLE_TILE:$ENABLE_TILE"
 Write-Host "Running: $buildCmd"
 Invoke-Expression $buildCmd
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)